Tests/UITests/TestApp/LLMFog/Account/AccountSetupHeader.swift

```diff
@@ -13,7 +13,7 @@ import SwiftUI
 
 struct AccountSetupHeader: View {
     @Environment(Account.self) private var account
-    @Environment(\.accountSetupState) private var setupState
+    @Environment(\._accountSetupState) private var setupState
```

Comment:
Seems like there is a deprecation warning on the `_accountSetupState` from `SpeziAccount`. Is there a way to work around this by using the "regular" `accountSetupState`?

Reply:
Actually I haven't spend to much time on that since it's just a UITest for the LLMFog. If you have a quick fix, I can definitely integrate that 👍

Reply:
I'm not familiar enough with SpeziAccount to provide a quick fix, I just strongly discourage writing new code with our already deprecated APIs. Please do a quick check of SpeziAccount (also the UI test apps or the SpeziTempalteApp) , there should be a better solution than the current one. As far as I remember, I mostly copied this code over from other apps that we have.